### ISSUE TITLE
Napari not responding when asked to label frames in deeplabcut

### ISSUE THREAD

**Issue by bjmoraes** [DeepLabCut/napari-deeplabcut#71]
Hi, I'm having trouble getting Napari to label frames in deeplabcut.

Everything goes well up to

```python
deeplabcut.label_frames('/Users/bjm/Desktop/videos_cr/chrv12-7/test-BJM-2023-05-19/config.yaml')
``` 

Napari opens but the rainbow wheel keeps turning with nothing happening and the icon reports "Application Not Responding", so I am not even able to select the frame files to label.

I'm on a MacBook Pro, 13-inch M1, 2020 running macOS Ventura 13.3.1(a).
I installed deeplabcut according to [developer instructions](https://deeplabcut.github.io/DeepLabCut/docs/installation.html) without issues.
Python is up to date:

```
Python 3.9.16 | packaged by conda-forge | (main, Feb  1 2023, 21:38:11) 
[Clang 14.0.6 ] on darwin
``` 

I also re-installed Napari for deeplabcut following its [website instructions](https://pypi.org/project/napari-deeplabcut/).

Help would be appreciated.
Cheers,
BM

**Comment by bjmoraes** [comment]
I will close this since I haven't had any answer for 20 days and I have worked around the issue without solving it.

**Comment by MMathisLab** [comment]
Hi @bjmoraes I can't reproduce the issue you describe and can label. I recommend to launch the GUI from the terminal vs and IDE.

**Comment by bjmoraes** [comment]
Hi @MMathisLab, thanks. That was the workaround I followed. Using the GUI all works fine; only when trying to label using Terminal commands do I get stalled. In the meantime, I have another issue using the Mac, iteration loss numbers are **huge** (no issue using a PC with the same data) but I may discuss this on a different thread.

**Comment by MMathisLab** [comment]
Hey! I just hit this Mac tensorflow metal issue on one dataset; you can change (lower!) the learning rate -- namely before starting to train and after creating training dataset go into the dlc-models, .. train folder, open pose_cfg.yaml and drop learning rate. @n-poulsen is working on automatically adjusting this for MacOS users now (I think ;), but that was the solution for me and him!

**Comment by bjmoraes** [comment]
Thank you! I will try that out and give you feedback on it :)
